internal/clientgen/javascript.go

```diff
@@ -80,6 +81,7 @@ func (js *javascript) Generate(buf *bytes.Buffer, appSlug string, md *meta.Data)
 	}
 
 	js.WriteString("// " + doNotEditHeader() + "\n\n")
+	js.WriteString("import fetch from \"cross-fetch\";\n")
```

Comment:
This adds an implicit dependancy which we've not documented in the generate command's help. We should add that if we go which this approach.

On the backend this was solvable without a new library, but by simply applying a bind against the default fetcher function. The issue was we defaulted in a way which apply the wrong prototype when calling the function

```typescript
return new Client(APIServer(), {
    auth: key,
    fetcher: fetch.bind(globalThis), // This is needed in cloudflare workers, otherwise it will report a TypeError: Illegal invocation
  })
```

So I wonder if we can fix this without needing `cross-fetch`